.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
     -   id: check-added-large-files
     -   id: mixed-line-ending
 -   repo: https://github.com/pycqa/flake8
-    rev: '5.0.0'
+    rev: '5.0.4'
     hooks:
```

Comment:
Please Change: This is a dependency update and might lead to problems further down the line - please submit this in a new PR so we can label and reproduce it easier.

Reply:
sorry, forgot to remove that file from committing. It was changed by some of the auto update invokes.